_includes/code/layout/pavlova/main.dart

```diff
@@ -38,47 +38,47 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   @override
   Widget build(BuildContext context) {
-    var titleText = new Container(
-      padding: new EdgeInsets.all(20.0),
-      child: new Text(
+    final Container titleText = new Container(
```

Comment:
I don't know about flutter specific lints, but this type is redundant with inference unless Container is a generic class (in which case I would still get rid of the RHS type and make it new Container<dynamic>(.